solutions/0000-0099/0049-group-anagrams-medium.md

```diff
@@ -72,3 +74,36 @@ public:
     }
 };
 ```
+</TabItem>
+<TabItem value="java" label="Java">
+<SolutionAuthor name="@ganajayant"/>
+
+```java
+class Solution {
+    private String sortString(String str) {
+        char[] chars = str.toCharArray();
+        java.util.Arrays.sort(chars);
+        return new String(chars);
+    }
+
+    public List<List<String>> groupAnagrams(String[] strs) {
+        HashMap<String, LinkedList<String>> hm = new HashMap<>();
+        for (int i = 0; i < strs.length; i++) {
+            if (hm.containsKey(sortString(strs[i]))) {
+                hm.get(sortString(strs[i])).add(strs[i]);
+            } else {
+                String sortedstring = sortString(strs[i]);
```

Comment:
put it outside the if block to make it clean. just replace what I sent you previously.